diagnosisApplication/src/main/interface_adapter/diagnosis/DiagnosisState.java

```diff
@@ -8,134 +8,41 @@ public class DiagnosisState {
 
     private int numDiagnoses = 0;
 
-    private  Integer diagnosis1Ranking = null;
-    private  String diagnosis1Name = null;
-    private  Integer diagnosis1Accuracy = null;
-    private  String diagnosis1Icd = null;
-    private  String diagnosis1ProfName = null;
-    private List<String> diagnosis1Specializations = null;
+    private HashMap<String, Object> diagnosis1;
+    private HashMap<String, Object> diagnosis2;
+    private HashMap<String, Object> diagnosis3;
 
-    private  Integer diagnosis2Ranking = null;
-    private  String diagnosis2Name = null;
-    private  Integer diagnosis2Accuracy = null;
-    private  String diagnosis2Icd = null;
-    private  String diagnosis2ProfName = null;
-    private List<String> diagnosis2Specializations = null;
-
-    private  Integer diagnosis3Ranking = null;
-    private  String diagnosis3Name = null;
-    private  Integer diagnosis3Accuracy = null;
-    private  String diagnosis3Icd = null;
-    private  String diagnosis3ProfName = null;
-    private List<String> diagnosis3Specializations = null;
-
-    public DiagnosisState() {
-
-    }
+    public DiagnosisState() {}
 
     public void setDiagnosis1(HashMap<String, Object> diagnosis1) {
-        this.diagnosis1Ranking = 1;
-        this.diagnosis1Name = (String) diagnosis1.get("name");
-        this.diagnosis1Accuracy = (Integer) diagnosis1.get("accuracy");
-        this.diagnosis1Icd = (String) diagnosis1.get("icd");
-        this.diagnosis1ProfName = (String) diagnosis1.get("profName");
-        this.diagnosis1Specializations = (List<String>) diagnosis1.get("specializations");
+        this.diagnosis1 = diagnosis1;
```

Comment:
Constructor to instantiate the first diagnosis output as a hash map rather than setting values for all the variables.